internal/services/keyvault/key_vault_managed_hardware_security_module_resource.go

```diff
@@ -275,7 +283,30 @@ func resourceArmKeyVaultManagedHardwareSecurityModuleUpdate(d *pluginsdk.Resourc
 		d.Set("security_domain_encrypted_data", encData)
 	}
 
-	return nil
+	if d.HasChange("network_acls") {
+		networkAclsRaw := d.Get("network_acls").([]interface{})
+		networkAcls := expandMHSMNetworkAcls(networkAclsRaw)
+		update := managedhsms.ManagedHsm{
+			Properties: &managedhsms.ManagedHsmProperties{
+				TenantId:    utils.String(d.Get("tenant_id").(string)),
```

Comment:
Is it necessary to include the tenant ID when it doesn't/cannot be changed?

Reply:
@manicminer I've tried to remove this line and got the following error:

```text
"BadRequest" Message="The request does not contain a valid 'TenantId' property."
```